templates/dataset.yaml

```diff
@@ -20,19 +20,55 @@ dataset:
+     baseline_electric_power_profile: 
+        variable_name:
+            name: variable_name 
         building_id:
-            name: building_id
-        availability:
-            name: availability
-            filepath: filepath/to/variable_name # [OPTIONAL] Supply filepath to dataset where variable can be found if different from <dataset>:<filepath> value
-        timestamps: timestamps
-        baseline_electric_power_profile: baseline_electric_power_profile
-        baseline_electricity_consumption_profile: baseline_electricity_consumption_profile
-        baseline_natural_gas_consumption_profile: baseline_natural_gas_consumption_profile
-        flexible_electric_power_profile: flexible_electric_power_profile
-        flexible_electricity_consumption_profile: flexible_electricity_consumption_profile
-        flexible_natural_gas_consumption_profile: flexible_natural_gas_consumption_profile
-        load_profile_peak_timestamp: load_profile_peak_timestamp
-        load_profile_valley_timestamp: load_profile_valley_timestamp
-...
\ No newline at end of file
+            name: building_id 
+        filepath: filepath/to/variable_name # [OPTIONAL] Supply filepath to dataset where variable can be found if different from <dataset>:<filepath> value
+     baseline_electricity_consumption_profile: 
+        variable_name:
+            name: variable_name 
+        building_id:
+            name: building_id 
+        filepath: filepath/to/variable_name # [OPTIONAL] Supply filepath to dataset where variable can be found if different from <dataset>:<filepath> value  
+     baseline_natural_gas_consumption_profile: 
+        variable_name:
+            name: variable_name 
+        building_id:
+            name: building_id 
+        filepath: filepath/to/variable_name # [OPTIONAL] Supply filepath to dataset where variable can be found if different from <dataset>:<filepath> value  flexible_electric_power_profile: flexible_electric_power_profile
+     flexible_electricity_consumption_profile: 
+        variable_name:
+            name: variable_name 
+        building_id:
+            name: building_id 
+        filepath: filepath/to/variable_name # [OPTIONAL] Supply filepath to dataset where variable can be found if different from <dataset>:<filepath> value 
+     flexible_natural_gas_consumption_profile: 
+        variable_name:
+            name: variable_name
+        building_id:
+            name: building_id 
+        filepath: filepath/to/variable_name # [OPTIONAL] Supply filepath to dataset where variable can be found if different from <dataset>:<filepath> value 
+     
+     
+     # does this refer to operation states? 
+     availability:
+        variable_name:
+            name: variable_name 
+        filepath: filepath/to/variable_name # [OPTIONAL] Supply filepath to dataset where variable can be found if different from <dataset>:<filepath> value 
+        
+     
+        # I think these are estimated variables, right?
```

Comment:
Yeah they are so maybe we don't need them in the field name mapping since they will not be fields in the dataset.

Reply:
Agree